src/discordTools/SetupGuildChannels.js

```diff
@@ -68,4 +68,19 @@ function addMissingTextChannels(client, guildId, parent) {
     else {
         events.setParent(parent.id);
     }
+    
```

Comment:
Channel id type needs to be added to createInstance file. possibly rename it to something better like teamchat instead because its used for ingame messages as well as discord messages